pkgs/servers/sabnzbd/default.nix

```diff
@@ -1,19 +1,27 @@
-{stdenv, fetchurl, python, pythonPackages, par2cmdline, unzip, unrar}:
+{stdenv, fetchurl, python, par2cmdline, unzip, unrar, p7zip, makeWrapper}:
 
-stdenv.mkDerivation rec {
-  version = "0.7.20";
+let
+  pythonEnv = python.withPackages(ps: with ps; [ pyopenssl python.modules.sqlite3 cheetah]);
+  path = stdenv.lib.makeBinPath [ par2cmdline unrar unzip p7zip ];
+in stdenv.mkDerivation rec {
+  version = "1.1.0";
   name = "sabnzbd-${version}";
 
   src = fetchurl {
-    url = "mirror://sourceforge/sabnzbdplus/SABnzbd-${version}-src.tar.gz";
-    sha256 = "0hl7mwgyvm4d68346s7vlv0qlibfh2p2idpyzpjfvk8f79hs9cr0";
+    url = "https://github.com/sabnzbd/sabnzbd/archive/${version}.tar.gz";
+    sha256 = "16srhknmjx5x2zsg1m0w9bipcv9b3b96bvb27fkf4dc2aswwcsc7";
   };
 
-  pythonPath = with pythonPackages; [ pyopenssl sqlite3 cheetah ];
-  buildInputs = with pythonPackages; [wrapPython];
-  inherit python par2cmdline unzip unrar;
+  buildInputs = [ pythonEnv makeWrapper ];
 
-  builder = ./builder.sh;
+  installPhase = ''
+    mkdir -p $out
+    cp -R * $out/
+    mkdir $out/bin
+    echo "${pythonEnv}/bin/python $out/SABnzbd.py \$*" > $out/bin/sabnzbd
+    chmod +x $out/bin/sabnzbd
+    wrapProgram $out/bin/sabnzbd --prefix PATH : ${path}
```

Comment:
Do you have to use `--prefix`? Would `--set` work? The only reason to avoid it is if the program would call a program the caller specified.

(disclaimer I'm not super familiar with sabnzbd)